src/policy/policy.h

```diff
@@ -22,6 +22,8 @@ static const unsigned int DEFAULT_BLOCK_MAX_SIZE = 2000000;
 static const unsigned int DEFAULT_BLOCK_MIN_TX_FEE = 1000;
 /** The maximum size for transactions we're willing to relay/mine */
 static const unsigned int MAX_STANDARD_TX_SIZE = 100000;
+/** The minimum size for transactions we're willing to relay/mine (1 input + 1 P2PKH output = 82 bytes) */
```

Comment:
bitcoin's comment specifies 1 segwit input. But a segwit input ins't any smaller than a non-segwit input is it? it's just a txid+index no?

Reply:
Segwit input is an input with an empty scriptSig... BUT you have a point here - I missed the fact that P2WPKH is 3 bytes smaller than P2PKH, so we should bump `MIN_STANDARD_TX_SIZE` I think, see 11aa04e.

Reply:
Good thing I mentioned it :) I was worried it might change the calculations, but didn't see how it would